<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeobaseSettings</class>
 <widget class="QDialog" name="GeobaseSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Геоподосновы - настройки</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>По двойному щелчку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbOnDblClick">
       <item>
        <property name="text">
         <string>Свойства</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Просмотр в Автокаде</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Выбор</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSelectBeh">
       <item>
        <property name="text">
         <string>Ячейка</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Строка</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSelectMode">
       <item>
        <property name="text">
         <string>Одна</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Несколько</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Чертежи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbDrawingShow">
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Имя файла</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ID - имя файла</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeobaseSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeobaseSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
